skimage/io/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+import numpy as np
+import imageio
```

Comment:
```suggestion
import imageio.v3 as iio
```

I'd generally recommend importing the version-major specific namespace (we also have `imageio.v2`) when using ImageIO. 

Both APIs will continue to exist going forward, but this way communicates to a fellow maintainer if they should use `v2.volread` or `v3.imread(..., index=...)` when modifying the file. It also has the nice side-effect of versioning scripts, because the expected version major is explicitly written into the import statement: "I can't remember which versions I used for this 3 years ago, but it says `import imageio.v3`, so I guess we could start with trying ImageIO v3.X", and it will just work. It also makes code more robust to future breaking changes to the API (in ImageIO v4+), which will exist in `import imageio.v4` should the need ever arise.

*Edit:* Also `iio.imread` looks way cooler than `imageio.imread` 😆